src/nunavut/_utilities.py

```diff
@@ -93,6 +93,18 @@ def test_truth(cls, ynd_value: "YesNoDefault", default_value: bool) -> bool:
     DEFAULT = 2
 
 
+@enum.unique
+class ResourceType(enum.Enum):
```

Comment:
This is part of the change to differentiate between support code that is only for serialization and support code that is required for generated PODs